packages/common/src/hardforks/muirGlacier.json

```diff
@@ -3,5 +3,13 @@
   "comment": "HF to delay the difficulty bomb",
   "url": "https://eips.ethereum.org/EIPS/eip-2384",
   "status": "Final",
-  "eips": []
```

Comment:
Had to remove this field, otherwise the difficulty bomb delay as reported by `param()` in Common yields the Constantinople one (probably a bug?).

Reply:
Hmm, interesting.